### ISSUE TITLE
Fail to encode long string

### ISSUE THREAD

**Issue by progsmile** [exthereum/abi#19]
Hello, dear developer!
Library fails when try to encode (and the same with decoding) long string

Here is sample:

```elixir
function_selector = %FunctionSelector{
  function: "setAccountFieldMainData",
  returns: nil,
  types: [:string, :string]
}
params = [{String.duplicate("1234567890", 10), "string"}]

ABI.encode(function_selector, params)
```

Suppose it's enough info to reproduce
Thank you for your time

**Comment by progsmile** [comment]
Found that limitation for string size is **64**

**Comment by hayesgm** [comment]
Dennis, sorry for the delay. I'll take a look at this. Any progress or are you blocked?

**Comment by hayesgm** [comment]
This should be fixed by #20. Let me know if you want me to cut a new release.

**Comment by progsmile** [comment]
We are not blocked, colleague has implemented own fix, but your's much cleaner  
Thanks
